runtime/v2/binary.go

```diff
@@ -86,6 +86,16 @@ func (b *binary) Start(ctx context.Context, opts *types.Any, onClose func()) (_
 	go func() {
 		defer f.Close()
 		if _, err := io.Copy(os.Stderr, f); err != nil {
+			// When using a multi-container shim, the fifo of the 2nd to Nth
+			// container will not be opened when the ctx is done. This will
+			// cause a "closed fifo" error that can be ignored.
+			select {
+			case <-ctx.Done():
+				if err.Error() == "reading from a closed fifo" {
```

Comment:
@darfux maybe we can update the fifo repo to define a error var so that we can use the var, instead of fixed string

Reply:
good idea! I'll work on it